### PR TITLE
🔨 Cyclical dependency toggle spacing

### PR DESCRIPTION
**Pull request by oddvernes**
This seems to fully work as of right now, but only without the  `'eds/css/spacing/shorthand'` transformer.
can be tested with import ` '@equinor/eds-tokens/css/variables.css'` or ` '@equinor/eds-tokens/css/variables-verbose.css'` in `eds-core-react\.storybook\preview.mjs`

### CODE REVIEW

File: packages/eds-tokens-build/src/index.ts [hunk 1]
```diff
@@ -68,6 +71,48 @@ const lightDarkTransform: StyleDictionary.Transform = {
   },
 }
 
+//MARK: SpaceToggleTransform
+const densitySpaceToggleTransform: StyleDictionary.Transform = {
```
Comment (torleifhalseth) [inline packages/eds-tokens-build/src/index.ts:75]: Great work. Could we a create separate file for this transform and maybe also for the other complex transforms we have? I think that would clean up this file and make it more readable.

File: packages/eds-tokens-build/src/index.ts [hunk 2]
```diff
@@ -228,7 +278,7 @@ StyleDictionary.registerTransform({
 
 const cssTransforms = [
   'name/cti/kebab',
-  'eds/css/spacing/shorthand',
+  //'eds/css/spacing/shorthand',
```
Comment (torleifhalseth) [inline packages/eds-tokens-build/src/index.ts:281]: What remains to make this work?

File: packages/eds-tokens/build/css/index-trimmed.css [hunk 3]
```diff
@@ -1,3 +1,3 @@
 @import 'color/theme-trimmed.css';
-@import 'spacing/comfortable-trimmed.css';
-@import 'spacing/spacious-trimmed.css';
+@import 'cyclical-dependency-setup.css';
```
Comment (torleifhalseth) [inline packages/eds-tokens/build/css/index-trimmed.css:2]: Could we name this file something like "density-setup"?

**Review by torleifhalseth — CHANGES_REQUESTED** [review]
Looks great. Added some comments.

File: packages/eds-tokens-build/src/index.ts (hunk 1 above)
Comment (oddvernes) [reply packages/eds-tokens-build/src/index.ts:75]: We'll do this in a sepereate branch later so we keep the git history for these changes, plus there is no time

File: packages/eds-tokens-build/src/index.ts (hunk 2 above)
Comment (oddvernes) [reply packages/eds-tokens-build/src/index.ts:281]: As discussed, we'll find a solution for this later as seperate tokens  (if we do it at all)

**Pull request merged**